lib/video/screenRecording/recorder.js

```diff
@@ -2,11 +2,21 @@
 const { isAndroidConfigured } = require('../../android');
 const AndroidRecorder = require('./android/recorder');
 const X11Recorder = require('./desktop/x11recorder');
+const FirefoxRecorder = require('./desktop/firefoxrecorder');
+const os = require('os');
 
-module.exports = function getRecorder(options) {
+module.exports = function getRecorder(options, browser) {
   if (isAndroidConfigured(options)) {
     return new AndroidRecorder(options);
   } else {
-    return new X11Recorder(options);
+    if (
```

Comment:
Let's not block based on OS, since we expect only experts to use this.  And this doesn't agree with the docstring (right?), so let's update that too.

Reply:
Blocking on OS for now until the other compositors are ready, but it might be ready now from Barret's patch so I'll have to test that out.  However, I do think this should be the default recorder over ffmpeg screen capture, especially for mach command usage.